podpac/core/coordinates/stacked_coordinates.py

```diff
@@ -222,15 +227,42 @@ def copy(self):
             Copy of the stacked coordinates, with provided properties and name.
         """
 
-        return StackedCoordinates(self._coords)
+        return StackedCoordinates(list(self._coords.values()))
 
     # ------------------------------------------------------------------------------------------------------------------
-    # standard methods, tuple-like
+    # standard dict-like methods
     # ------------------------------------------------------------------------------------------------------------------
+    def keys(self):
```

Comment:
I probably wouldn't include `keys`, `items`, `get` and `update` in the public StackedCoordinates interface. It does make sense to have `values` in the public interface, though I would probably leave `iter` with the original meaning (now implemented as `return self._coords.values()`) and then you can use that instead `values`. But I think that's a small thing. I like

```
c = StackedCoordinates([<lat>, <lon>])
lat, lon = c
```

But I can see that this might be clearer.

```
c = StackedCoordinates([<lat>, <lon>])
lat, lon = c.values()
```